terraform/modules/kubernetes/variables.tf

```diff
@@ -44,11 +44,41 @@ variable "redis_cache_sku_name" {
   default = "Standard"
 }
 
+variable "replicas_per_master" {
+  type        = number
+  description = "Number of replicas per master cache node"
+  default     = 1
+}
+
+variable "redis_replica_zones" {
+  type        = list(string)
+  description = "List of availability zones for replicas"
+  default     = ["1","2"]
+}
+
 variable "redis_queue_capacity" {
   type    = number
   default = 1
 }
 
+variable "redis_backup_enable" {
+  type    = bool
+  default = false
+
+}
+
+variable "redis_backup_frequency" {
+  type    = string
+  description = "backup frequency in minutes"
+  default = "360"
+}
+
+variable "redis_snapshot_count" {
+  type    = string
+  description = "number of snapshots to keep"
+  default = "1"
+
+}
```

Comment:
Are these needed now we are using AOF?

Reply:
not really but should we not leave them in there in case we decide to do back on rdb back up